attribute-service-api/src/main/proto/org/hypertrace/core/attribute/service/v1/attribute_metadata.proto

```diff
@@ -103,6 +103,7 @@ message AttributeMetadata {
   string scope_string = 17;
   bool internal = 18;
   bool custom = 19;
+  string description = 20;
```

Comment:
```suggestion
  optional string description = 20;
```

Reply:
This change may not be required. Not setting the description value will not throw an error. It will just be set to it's default value, i.e., an empty string. 
Is there any other reason for having it as optional?

Reply:
I was mostly thinking from client’s perspective, if they should be handling empty strings (they would anyways have to I guess, even if the field is set), or testing for the presence of description field

We should be good without optional too

Reply:
In both the cases a) old record without description and b) new record created without a description, the object returned by graphql will have an empty string for description.